Havit/Havit/Screens/CategoryContents/View/Cell/CategoryFilterCollectionViewCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CategoryFilterCollectionViewCell.swift
+//  Havit
+//
+//  Created by 박예빈 on 2022/01/12.
+//
+
+import UIKit
+
+final class CategoryFilterCollectionViewCell: UICollectionViewCell {
+    static var cellID = "CategoryFilterCollectionViewCell"
+    
+    var filterNameLabel: UILabel!
```

Comment:
p2;

이것도 
```suggestion
    var filterNameLabel = UILabel()
```
로 해주세요! 밑에 `filterNameLabel = UILabel()`는 지우면 될 거 같습니다!

Reply:
수정했습니다! 감사합니당❤️